template-helpers/hbs-helpers.js

```diff
@@ -9,6 +9,9 @@ const log = mozlog("template-helpers/hbs-helpers");
 
 function getSupportedLocales(args) {
   if (args.data) {
+    if (args.data.root.supportedLocales) {
+      return args.data.root.supportedLocales;
+    }
```

Comment:
Nested `if` block smells like we've got some inconsistency here? Maybe a follow-up, but can we sniff out why we sometimes have `supportedLocales` in `args.data.root` and sometimes in `args.data.root.req`?

Reply:
It's because `supportedLocales` is sent directly to the email templates (`args.data.root.supportedLocales`) whereas we pluck it from the req in page templates.

Reply:
Ah yeah. Hmm ... I'm sure there's some fancy JS syntax to use here, but they would probably just make it harder to read.